app_user.go

```diff
@@ -21,59 +30,344 @@ func newAppUser(app App) *appUser {
 // Get gets a user using their userID
 // returns user on success, error on failure
 func (a *appUser) Get(userID string) (*PassageUser, error) {
-	return a.app.GetUser(userID)
+	message := "failed to get Passage User"
```

Comment:
i think in all of the functions, the message doesn't really need to state "what" happened. since the error is usually handled directly after the function call, that should be sufficient to figure out the "what."

instead, if the error message were to focus on the "why" it could be more helpful.

mapping `message` to something like `res.JSONX.Error` would provide the error message directly from our API explaining why the error occurred.

Reply:
made this change